workshops/lib/twilio-basic/signup.md

```diff
@@ -105,14 +105,11 @@ For your convenience, we have provided a credit card to signup with below. You
 may use your own card as well (your card will not be charged if you turn off
 automatic recharging).
 
-| Field              | Value                               |
-| -------------------| ----------------------------------- |
-| Credit Card Number | `374327384095936`                   |
-| Expires            | `10/2021`                           |
-| CVV                | `9263`                              |
-| Cardholder Name    | `Jonathan Leung`                    |
-| Street Address     | `200 Haverford Road, Wynnewood, PA` |
-| Zip Code           | `19096`                             |
+To use our credit card, just email Jonathan at jonathanleung1337@gmail.com and
```

Comment:
@zachlatta I removed the reference to Jonathan.

@paked shipping this for now without a hackEDU address because it's a bit of a hassle to setup at the moment.